partitioner/src/main/java/io/confluent/connect/storage/partitioner/DefaultPartitioner.java

```diff
@@ -42,7 +42,7 @@
   @Override
   public void configure(Map<String, Object> config) {
     this.config = config;
-    delim = (String) config.get(StorageCommonConfig.DIRECTORY_DELIM_CONFIG);
+    delim = getDirectoryDelimiter(config);
```

Comment:
Made this to a method because this config is gotten in several places, and I was getting a NPE when the value wasn't supplied.